src/public/style.css

```diff
@@ -55,3 +55,176 @@ a {
 a:hover {
   color: #363636;
 }
+
+#contact-form {
```

Comment:
CSS is out of scope for #10. Try to stick to the requirements of the issue you are working on. Taking on extra work that is out of scope (often referred to as "boy-scouting") is generally frowned upon when you're working on a team.